tests/unit/fem/test_estimator.cpp

```diff
@@ -618,7 +655,6 @@ TEST_CASE("Kelly Error Estimator on 3D NCMesh",
       {
          REQUIRE(local_errors(i) == MFEM_Approx(0.0));
       }
-      REQUIRE(estimator.GetTotalError() == MFEM_Approx(0.0));
```

Comment:
Why is the perfect approximation no longer giving zero error? If it's no longer perfect, these sections might need renaming?

Reply:
It was also not exactly zero before, but something in the order 1e-17, because of the bug. Now it is in the order of 1e-14, which is above the error check threshold. I am pretty sure this is due to the slight numerical differences being added up.